integration-testing/election-manager/cypress/support/auth.ts

```diff
@@ -43,5 +45,5 @@ export function mockCardRemoval(): void {
 }
 
 export function logOut(): void {
-  cy.request('POST', 'http://localhost:3000/api/logOut', {});
+  cy.request('POST', methodUrl('logOut', 'http://localhost:3000/api'), {});
```

Comment:
I know this is super rudimentary, and we'd ideally use the `Api` type to guarantee that the specified method is actually present in the API, but I wanted to time box. Made an issue for an improved version: https://github.com/votingworks/vxsuite/issues/2969

Reply:
Sounds good!